VideoLocker/res/layout/row_course_outline_list.xml

```diff
@@ -104,6 +104,7 @@
             android:background="@android:color/transparent"
             android:paddingEnd="5dp"
             android:paddingRight="5dp"
+            android:gravity="center"
```

Comment:
Since you have set the gravity to be centralized in this layout, you can remove the `android:gravity="center_vertical"` and `android:layout_centerVertical="true"` attributes from the `no_of_videos` view. Note that this `LinearLayout` and it's siblings also have these useless layout gravity attributes, when their parent already has it's gravity defined as vertically centered. We can also get rid of all the layout attributes for `RelativeLayout` in these views, as there is no `RelativeLayout` present in this layout.